pkg/volume/csi/csi_attacher.go

```diff
@@ -351,18 +351,6 @@ func (c *csiAttacher) MountDevice(spec *volume.Spec, devicePath string, deviceMo
 	}
 
 	err = saveVolumeData(dataDir, volDataFileName, data)
-	defer func() {
```

Comment:
If csi.NodeSupportsVolumeMountGroup(ctx) returns an unwrapped error, will the vol_data file be left behind?

Reply:
It is fixed after I applied gnufied commit.